code/modules/hivemind/machines.dm

```diff
@@ -440,17 +441,18 @@
 /obj/machinery/hivemind_machine/mob_spawner
 	name = "assembler"
 	desc = "This cylindrical machine has lights around a small portal. The sound of tools comes from inside."
-	max_health = 160
+	max_health = 260
+	resistance = RESISTANCE_IMPROVED //similary to the turret, primary defence of the hive
 	icon_state = "spawner"
-	cooldown_time = 30 SECONDS
-	spawn_weight  =	45
+	cooldown_time = 25 SECONDS  //shorter cooldown
+	spawn_weight  =	50 //more common than earlier
 	var/mob_to_spawn
-	var/mob_amount = 3
+	var/mob_amount = 4
 
 /obj/machinery/hivemind_machine/mob_spawner/Initialize()
 	..()
-	mob_to_spawn = pick(/mob/living/simple_animal/hostile/hivemind/stinger, /mob/living/simple_animal/hostile/hivemind/bomber)
-
+	mob_to_spawn = pick(/mob/living/simple_animal/hostile/hivemind/stinger, /mob/living/simple_animal/hostile/hivemind/bomber, /mob/living/simple_animal/hostile/hivemind/hiborg)
```

Comment:
This is a whole lot of buffs for the spawner, and with the buffed borg it should probably be a lower pickweight for the borg to spawn instead of them all having the same chance.

Reply:
fixed, assembler now uses pickweight (hiborg being the lowest of em) and can spawn multiple types of mobs instead of sticking to just one.